.github/ISSUE_TEMPLATE/mentorship.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Mentorship request
+about: Interested in getting involved, but want some guidance? Submit a mentorship
+request!
+title: Mentorship request for <your-name>
+labels: mentorship
+assignees: ''
+
+---
+
+<!--
+If you're new to the project, welcome!
+
+In addition to filling out this mentorship request, consider joining the
+in-toto slack community and introducing yourself!
+
+https://app.slack.com/client/T08PSQ7BQ/CM46K2VT2
```

Comment:
Trying to open this in a private tab asks me to sign in, with even the workspace URL not filled in.

Reply:
```suggestion

```